README.md

```diff
@@ -42,4 +45,5 @@ yields:
    password=bar
    ```
 
-2. Publish a new release with `sbt publishSigned`
+Any commits to master should also trigger a circleci build that will do the sbt publishing for you
```

Comment:
You can remove the `1.` section entirely, since we're not using sonatype any longer for this repo.